ControlMeasurements/Migrations/20190211072151_Measurement.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace ControlMeasurements.Migrations
 {
-    public partial class Measurements : Migration
+    public partial class Measurement : Migration
```

Comment:
Nazwa migracji nie ma specjalnego znaczenia :) To tylko skrypt, ktory Ci zrobi zmiany na bazie